src/server.ts

```diff
@@ -2,16 +2,49 @@ import { ApolloServer } from "@apollo/server";
 import { startStandaloneServer } from "@apollo/server/standalone";
 import fs from "fs";
 import { resolvers } from "./resolvers.js";
+import AnalyticsDataSource from "./datasources/analytics.js";
+import { config } from "dotenv";
+
+if (process.env.DB_CONN_STRING == null) config({ path: ".env.local" });
 
 const typeDefs = fs.readFileSync("src/schema.graphql", "utf8");
 
-const server = new ApolloServer({
-  typeDefs,
-  resolvers,
-});
+export interface ContextValue {
+  dataSources: {
+    analytics: AnalyticsDataSource;
+  };
+}
+
+export const startServer = async () => {
+  const analyticsDS = new AnalyticsDataSource(process.env.DB_CONN_STRING);
+  await analyticsDS.initialize();
```

Comment:
Before the server announces itself ready (line 47), connection to MongoDB should be established first, and output the log from https://github.com/chohanbin/flywheel-data-service/pull/4/files#r1630251591